### ISSUE TITLE
CoT failed to find committer data for commit

### ISSUE THREAD

**Issue by bhearsum** [mozilla-releng/scriptworker#477]
I noticed this while investigating https://github.com/mozilla-mobile/fenix/pull/17480. On that PR, there are _push_ event tasks (I assume they were created when the branch was pushed, not when the PR was opened). In any case, it fails to find commit data for reasons I haven't been able to figure out:
```
2021-01-15T10:40:16 CRITICAL - Chain of Trust verification error!
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1965, in verify_chain_of_trust
    await verify_task_types(chain)
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1724, in verify_task_types
    await valid_task_types[task_type](chain, obj)
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1648, in verify_parent_task
    await verify_parent_task_definition(chain, link)
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1519, in verify_parent_task_definition
    jsone_context, tmpl = await get_jsone_context_and_template(chain, parent_link, decision_link, tasks_for)
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1489, in get_jsone_context_and_template
    jsone_context = await populate_jsone_context(chain, parent_link, decision_link, tasks_for)
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1259, in populate_jsone_context
    jsone_context.update(await _get_additional_github_push_jsone_context(decision_link))
  File "/app/lib/python3.8/site-packages/scriptworker/cot/verify.py", line 1162, in _get_additional_github_push_jsone_context
    committer_login = commit_data["committer"]["login"]
TypeError: 'NoneType' object is not subscriptable
```

**Comment by escapewindow** [comment]
That PR was created from `mergify/bp/releases_v85.0.0/pr-16786` .
I'm wondering if this is an example of https://bugzilla.mozilla.org/show_bug.cgi?id=1680064 : taskcluster splits by `/` and might assume we're looking at, say, the `releases_v85.0.0` branch rather than the mergify branch on top of it.

**Comment by escapewindow** [comment]
```
committer_login = commit_data["committer"]["login"]
TypeError: 'NoneType' object is not subscriptable
```

It looks like `committer` isn't set.
I'm guessing this means @irios doesn't have full git metadata (https://github.com/mozilla-mobile/fenix/commit/7a4cdd5de3e46bc8f1c49a5389ca95926a131a24 had `isabelrios@MBP-de-isabel-2.home` as the email; does that mean it isn't properly configured?) I'm also guessing that means that pushing a new change with a valid login person may fix it. I tried pushing the space-removal change as me.

I pushed the exact same change as me, https://github.com/mozilla-mobile/fenix/commit/57e7297c5aeca2a2dffc16e690a72009294c77fe . That passed CoT.
